src/bin/casr-afl.rs

```diff
@@ -0,0 +1,274 @@
+extern crate anyhow;
+extern crate casr;
+extern crate clap;
+extern crate goblin;
+#[macro_use]
+extern crate log;
+
+use anyhow::{bail, Context};
+use clap::{App, Arg};
+use simplelog::*;
+
+use casr::error;
+
+use std::fs;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+#[derive(Debug, Clone, Default)]
+struct AflCrashInfo {
+    /// Path to crash input.
+    pub path: PathBuf,
+    /// Target command line args.
+    pub target_args: Vec<String>,
+    /// Stdin.
+    pub is_stdin: bool,
+    /// ASAN.
+    pub is_asan: bool,
+}
+
+fn main() -> error::Result<()> {
```

Comment:
We need to remove custom error types like Casr-san/Casr-cluster, etc. Now all anyhow errors cast to Casr-cluster error. This leads to wrong message:

Error: Cluster("No such file or directory (os error 2). Couldn't launch casr-san. ")